src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -22,6 +23,29 @@ class SignInModal extends React.Component {
     }
   }
 
+  renderDSLogonDowntimeBanner = (status) => {
+    if (status === 'down') {
+      return (
+        <div className="downtime-notification row">
+          <div className="columns small-12">
+            <div className="form-warning-banner">
+              <AlertBox
+                headline="DS Logon isn't working quite right"
+                content="If you're having trouble signing in to Vets.gov using your DS Logon username and password, please try again later. Or, you can try signing in with your My HealtheVet username and password or through ID.me."
+                isVisible
+                status="warning"/>
+              <br/>
+            </div>
+          </div>
+        </div>
+      );
+    }
+
+    return (
+      <div></div>
+    );
```

Comment:
This is another point of improvement for the DowntimeNotifications; it shouldn't force you to return an empty element if nothing needs to be done.